flake8_quotes/__init__.py

```diff
@@ -29,13 +30,35 @@ class QuoteChecker(object):
     def __init__(self, tree, filename='(none)', builtins=None):
         self.filename = filename
 
+    @staticmethod
+    def _register_opt(parser, *args, **kwargs):
+        """
+        Handler to register an option for both flake8 version 3 and version 2.
+
+        This is based on:
+        http://flake8.pycqa.org/en/latest/plugin-development/cross-compatibility.html
+
+        It only supports `parse_from_config` from the original function and it
+        uses the `Option` object returned to get the string.
+        """
+        try:
+            # Flake8 3.x registration
```

Comment:
We should be consistent about saying "Flake8 version 3" or "Flake8 3.x". Please choose one variant and use that consistently. My personal preference is `Flake8@3` but the current ones are good as well.